rick-and-morty/Source/Data/Episodes/DTO/EpisodeDTO.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  EpisodeDTO.swift
+//  rick-and-morty
+//
+//  Created by Carlos Morgado on 15/9/23.
+//
+
+import Foundation
+
+struct EpisodeDTO: Decodable {
+    let id: Int
+    let name: String
+    let air_date: String
```

Comment:
Intenta no poner variables así porque no encaja con el estilo de Swift. Obviamente lo has puesto para que encaje con la definición de la API, pero puedes hacer esto para evitarlo: https://stackoverflow.com/questions/69804293/when-to-use-codingkeys-in-decodableswift